Jenkinsfile

```
@@ -9,7 +9,7 @@ properties([[$class: 'BuildDiscarderProperty',
  *  https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
  */
 Map branches = [:]
-['maven'/* TODO broken , 'maven-windows'*/].each {label ->
+['maven-11'/* TODO broken , 'maven-windows'*/].each {label ->
```

BTW - would it not make sense to keep java8 as an option here?  Jenkins still supports java8 for the LTS versions and if there are any security issues in Jetty that need an upgrade it would be nice to know we have tested winstone in a Java8 environement (and for non LTS also until java11 becomes a strict requirement)

```suggestion
['maven-11', 'maven' /* TODO broken , 'maven-windows'*/].each {label ->
```